app/src/main/java/org/jellyfin/mobile/player/PlayerViewModel.kt

```diff
@@ -450,6 +458,19 @@ class PlayerViewModel(application: Application) : AndroidViewModel(application),
         }
     }
 
+    override fun onPlayerError(error: PlaybackException) {
+        if (error.cause is MediaCodecDecoderException && !fallbackPreferExtensionRenderers) {
+            Timber.e(error.cause, "Decoder failed, attempting to restart playback with decoder extensions preferred")
+            playerOrNull?.run {
+                removeListener(this@PlayerViewModel)
+                release()
+            }
+            fallbackPreferExtensionRenderers = true
```

Comment:
And when will we put this back to false? 🤔

Reply:
We don't, but you normally don't play a lot of stuff in a row, and if you do, it's likely the same codec (e.g. a show).